api/spec/v1.yaml

```diff
@@ -2100,6 +2104,34 @@ components:
       description: |
         A runtime transaction.
 
+    RuntimeTransactionEncryptionEnvelope:
+      type: object
+      required: [format, public_key, data_nonce, data, result_nonce, result]
+      properties:
+        format:
+          type: string
```

Comment:
Introduce an openapi enum please. Warren introduced it as `common.CallFormat` in #407; you can refer to it with `x-go-type` (see existing examples in v1.yaml).

@pro-wh what do we do for `CallFormatPlain`-encoded txs? (https://github.com/oasisprotocol/oasis-sdk/blob/c823194ed60ad7a922812e6b803d8ff8b9f78c45/client-sdk/go/callformat/callformat.go#L33-L33) Do we not support them, or do we support them because that's just how a nonencrypted tx is always represented, or something else?

Reply:
oh wait, it would be possible to determine that by examining the body